app/models/account_request.rb

```diff
@@ -40,28 +44,25 @@ def identity_token
     JWT.encode({ account_request_id: id }, Rails.application.secrets[:secret_key_base], 'HS256')
   end
 
+  # @return [Boolean]
   def confirmed?
-    confirmed_at.present?
+    pending? || approved?
   end
 
+  # @return [Boolean]
   def processed?
     organization.present?
   end
 
-  def approved?
-    "Approved" if confirmed? && Organization.present?
-  end
-
-  def pending_approval?
-    "Pending Approval" if confirmed? && Organization.blank?
-  end
-
-  def requested?
-    "Requested" unless confirmed?
+  def approve!
```

Comment:
Ahh, I was a bit confused with `approve!` in this case. Could we find a better enum state to indicate that approval is pending and isn't approved?

Reply:
Sorry about that - I based the names of the enums off existing methods on the `account_request` model. More than happy to change it.

Reply:
All good :)